src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -128,14 +189,15 @@ const TopicFragment = ({
           __renderDiscoveryItem(FROM_FOLLOWED_TOPIC, 'followedTopicDiscovery', item, index)
         )}
 
-        {unfollowedTopic.length > 0 && followedTopic.length > 0 && (
+        {route.name !== 'Followings' && unfollowedTopic.length > 0 && followedTopic.length > 0 && (
           <View style={styles.unfollowedHeaderContainer}>
             <Text style={styles.unfollowedHeaders}>{StringConstant.discoveryMoreTopics}</Text>
           </View>
         )}
-        {unfollowedTopic.map((item, index) =>
-          __renderDiscoveryItem(FROM_UNFOLLOWED_TOPIC, 'unfollowedTopicDiscovery', item, index)
-        )}
+        {route.name !== 'Followings' &&
+          unfollowedTopic.map((item, index) =>
+            __renderDiscoveryItem(FROM_UNFOLLOWED_TOPIC, 'unfollowedTopicDiscovery', item, index)
+          )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The rendering of the "More Topics" header and the list of unfollowed topics is now conditional on the route name. This change seems to be necessary for the new feature.
```diff
- {unfollowedTopic.length > 0 && followedTopic.length > 0 && (
-    <View style={styles.unfollowedHeaderContainer}>
-      <Text style={styles.unfollowedHeaders}>{StringConstant.discoveryMoreTopics}</Text>
-    </View>
-  )}
- {unfollowedTopic.map((item, index) =>
-   __renderDiscoveryItem(FROM_UNFOLLOWED_TOPIC, 'unfollowedTopicDiscovery', item, index)
- )}
+ {route.name !== 'Followings' && unfollowedTopic.length > 0 && followedTopic.length > 0 && (
+    <View style={styles.unfollowedHeaderContainer}>
+      <Text style={styles.unfollowedHeaders}>{StringConstant.discoveryMoreTopics}</Text>
+    </View>
+  )}
+ {route.name !== 'Followings' &&
+   unfollowedTopic.map((item, index) =>
+     __renderDiscoveryItem(FROM_UNFOLLOWED_TOPIC, 'unfollowedTopicDiscovery', item, index)
+   )}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->